meson.build

```diff
@@ -25,16 +20,31 @@ dep_hidapi = dependency(hidapi)
 dep_threads = dependency('threads')
 
 deps = [
-	meson.get_compiler('c').find_library('m', required : false), #-lm
-	dependency('threads') #pthread
+	meson.get_compiler('c').find_library('m', required: false),  # -lm
```

Comment:
For consistency, you could put this above and have 

```
dep_libm = meson.get_compiler('c').find_library('m', required: false)

deps = [
	dep_threads,
	dep_libm
]
```

Reply:
Done, thanks for the review! :D